src/components/_LuxUniversityAccessibility.vue

```diff
@@ -59,7 +59,6 @@ export default {
     a {
       color: var(--color-white);
       background: var(--color-gray-100);
-      &:hover,
```

Comment:
On hover in dark theme we would want
&:hover {
text-decoration-color: var(--color-princeton-orange-on-black);
}